utils/monitor_soap.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+# 
+# This Script can trigger the soap run and monitor it until completion.
```

Comment:
can you rename this script name as it is capable of trigger the soap run and monitor it ?

Reply:
It can do both things sir!
Can I name it as `run_and_monitor_soap.sh`??

Reply:
yes